webapp/src/components/link_tooltip/tooltip.css

```diff
@@ -13,11 +13,16 @@
 }
 
 /* Github Tooltip */
+.github-tooltip {
+    z-index: 1070;
```

Comment:
@asaadmahmood I've tested these changes and they do **not** resolve https://mattermost.atlassian.net/browse/MM-25904. Actually, I think this _has_ to be fixed in the webapp.

Reply:
@hanzei Was talking about a different z-index. And maybe that z-index lives in the webapp.